File: Ser2022/Store/Store/Store.Web/Modules/Store/Shippers/ShippersForm.cs

namespace Store.Store.Forms
{
    using Serenity.ComponentModel;
    using System;

    [FormScript("Store.Shippers")]
    [BasedOnRow(typeof(Entities.ShippersRow), CheckNames = true)]
    public class ShippersForm
    {
        public String CompanyName { get; set; }
        public String Phone { get; set; }
    }
}

File: Ser2023/Company/Company.Web/Modules/ServerTypes/Store/ShippersForm.ts
import { StringEditor, PrefixedContext } from "@serenity-is/corelib";
import { initFormType } from "@serenity-is/corelib/q";

export interface ShippersForm {
    CompanyName: StringEditor;
    Phone: StringEditor;
}

export class ShippersForm extends PrefixedContext {
    static formKey = 'Store.Shippers';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!ShippersForm.init)  {
            ShippersForm.init = true;

            var w0 = StringEditor;

            initFormType(ShippersForm, [
                'CompanyName', w0,
                'Phone', w0
            ]);
        }
    }
}